components/mintmaker/base/mintmaker-config.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: renovate-config
+  namespace: mintmaker
+data:
+  renovate.json: |
+    module.exports = {
+      "onboarding": false,
+      "requireConfig": "ignored",
+      "platform": "github",
+      "platformCommit": true,
+      "autodiscover": false,
+      "tekton": {
```

Comment:
I think we should use the same `tekton` manager config from https://gist.github.com/qixiang/915389b2c26b8ca5dbe479e2a5038bb0, that's the exact `tekton` manager config from build-service.

Reply:
But should we use the same? To be honest some of their configuration don't seem very reasonable to me (or we are unaware of the reasons behind it). I wouldn't keep it as it is. We can discuss about it in more details in the future if you want, but I think that this as an initial version should be ok.

Reply:
> But should we use the same?

I believe the answer is yes, once MintMaker is deployed, there are actually two Renovate, one from build-service, one from MintMaker, and both are running the Tekton manager (with same account), we should not expect different behavior between the two Renovate.

Reply:
These are the settings I don't like and why:
* dependencyDashboard: the dependency dashboard should be enabled in my opinion
* branchName: I don't think we should specify a different branch, also because RHEL wants to specify a branch -- perhaps they don't care about tekton, but I don't want to maintain managers that are that different
* commitBody: this should be already be added if the commit is signed, which is something we have to do
* commitMessageTopic: I dont' see why change that
* recreateWhen and rebaseWhen: I don't think we want to change that, we want to decide that when we'll have the on-demand